eip-0020.md

```diff
@@ -0,0 +1,196 @@
+# Ergo Pay: an interaction protocol between mobile wallet and dApp
+
+* Author: @aslesarenko, @MrStahlfelge
+* Status: Proposed
+* Created: 18-August-2021
+* License: CC0
+* Forking: not needed 
+
+## Contents
+- [Description](#description)
+- [Background And Motivation](#background-and-motivation)
+- [ErgoPay Interaction Protocol](#ergopay-interaction-protocol)
+- [Data Formats](#data-formats)
+- [Implementation in Mobile Wallet](#implementation-in-mobile-wallet)
+- [Implementation in dApp](#implementation-in-dapp)
+- [Benefits for dApps](#benefits-for-dapps)
+- [Benefits for Mobile Wallets](#benefits-for-mobile-wallets)
+
+## Description 
+This EIP defines a standard for cross-platform interaction between an online dApp
+and a mobile wallet for creating, signing and sending Ergo transactions.
```

Comment:
I'm not sure I get the idea, please clarify.

Reply:
When we restrict this to transmit the information via QR code, this can be only used when using both desktop and mobile device together. When used on a single mobile device or on a desktop, we need another way to start the process. This can be done with said link ergopay://signingrequest?data= which defines a new URI scheme to hand the data over to an application. Handlers can be registered for this [on Desktop](https://medium.com/swlh/custom-protocol-handling-how-to-8ac41ff651eb) and [on mobile](https://stackoverflow.com/a/11421251/7487013)

Reply:
@MrStahlfelge sounds good, please add the necessary description in the text.